requirements.txt

```diff
@@ -1,4 +1,4 @@
-django >= 1.7.0
+django >= 1.8.0
```

Comment:
I prefer specific to have specific requirements for testing - thus we can clearly see for what version it definitely works.